tpchgen/src/random.rs

```diff
@@ -299,6 +300,7 @@ pub struct RandomAlphaNumeric {
     inner: RowRandomInt,
     min_length: i32,
     max_length: i32,
+    buffer: Vec<u8>,
```

Comment:
Reuses the same allocation across calls to avoid having to copy bytes again into String